src/Services/Catalog/Catalog.API/Extensions/CatalogItemExtensions.cs

```diff
@@ -6,9 +6,12 @@ public static void FillProductUrl(this CatalogItem item, string picBaseUrl, bool
         {
             if (item != null)
             {
-                item.PictureUri = azureStorageEnabled
-                   ? picBaseUrl + item.PictureFileName
-                   : picBaseUrl.Replace("[0]", item.Id.ToString());
+                // https://www.linkedin.com/feed/update/urn:li:activity:6841055191211491328/
```

Comment:
Do we really need this link to the Linkedin post here? It didn't answer the question what is better to use